docker/main/Dockerfile

```diff
@@ -196,6 +196,8 @@ EXPOSE 8555/tcp 8555/udp
 
 # Configure logging to prepend timestamps, log to stdout, keep 0 archives and rotate on 10MB
 ENV S6_LOGGING_SCRIPT="T 1 n0 s10000000 T"
+# Do not fail on long-running download scripts
+ENV S6_CMD_WAIT_FOR_SERVICES_MAXTIME=0
```

Comment:
we shouldn't set this in the main dockerfile, we should set it in the ROCM dockerfile

Reply:
I see now why you added this, because you're planning on adding download support for the built in detectors.